release note for 0.9.1 (#918)
spiceai/spiceai: docs/release_notes/v0.9.1-alpha.md

> The v0.9.1 release adds `flightsql` data connector and `sqlite` data store. Improved the usability by adding more `spice` CLI commands.

Reviewer: ```suggestion
The v0.9.1 release focused on stability, bug fixes, and usability by adding `spice` CLI commands for listing Spicepods (`spice pods`), Models (`spice models`), Datasets (`spice datasets`), and improved status (`spice status) details. In addition, the Arrow Flight SQL (`flightsql`) data connector and SQLite (`sqlite`) data store were added. 
```

## Highlights in v0.9.1-alpha

**FlightSQL data connector**: [Arrow Flight SQL](https://arrow.apache.org/docs/format/FlightSql.html) can now be used as a connector for federated SQL query.

> **SQlite data backend**: [SQLite](https://www.sqlite.org/) can now be used as a data store for acceleration.

Reviewer: ```suggestion
**SQLite data backend**: [SQLite](https://www.sqlite.org/) can now be used as a data store for acceleration.
```

## Contributors

- @phillipleblanc
- @mitchdevenport
- @Jeadie
- @ewgenius
- @sgrebnov
- @y-f-u
- @lukekim

## New in this release

- **Adds** FlightSQL data connector (`flightsql`).

> - **Adds** SQlite data store.

Reviewer: ```suggestion
- **Adds** SQLite data store (`sqlite`).
```

> - **Adds** SQLite data store (`sqlite`).

Reviewer 2: Might want to mention support for in-memory and file based since not all data stores support both

> - **Adds** support for date, varchar, bpchar, primitive list type in Postgres.

Reviewer: ```suggestion
- **Adds** support for date, varchar, bpchar, and primitive list types for the PostgreSQL data connector and data store.
```

- **Adds** `spice pods`, `spice status`, `spice datasets`, and `spice models` CLI commands.

> - **Adds** `v1/spicepods` API.

Reviewer: ```suggestion
- **Adds** `GET /v1/spicepods` API for listing loaded Spicepods.
```

> - **Adds** `spiced` docker build and release.

Reviewer: ```suggestion
- **Adds** `spiced` Docker CI build and release.
```

> - **Adds** e2e tests for release installation and local acceleration.

Reviewer: ```suggestion
- **Adds** E2E tests for release installation and local acceleration.
```

> - **Fixes** `spice sql` REPL panics when query result is too large.

Reviewer: @y-f-u Please add the Issue # reference for bug fixes.

Reviewer 2: Also, maybe just say `bug fixes for spice sql` and then list them

> - **Adds** E2E tests for release installation and local acceleration.

Reviewer 3: Please add
- **Adds** E2E tests and instructions to run basic [TPC-H benchmark](https://www.tpc.org/tpch/) tests

> - **Fixes** `--access-secret`

Reviewer 2: What does it fix?

> The v0.9.1 release focused on stability, bug fixes, and usability by adding `spice` CLI commands for listing Spicepods (`spice pods`), Models (`spice models`), Datasets (`spice datasets`), and improved status (`spice status) details. In addition, the Arrow Flight SQL (`flightsql`) data connector and SQLite (`sqlite`) data store were added. 

Reviewer: ```suggestion
The v0.9.1 release focused on stability, bug fixes, and usability by adding `spice` CLI commands for listing Spicepods (`spice pods`), Models (`spice models`), Datasets (`spice datasets`), and improved status (`spice status`) details. In addition, the Arrow Flight SQL (`flightsql`) data connector and SQLite (`sqlite`) data store were added. 
```
